src/Vault.sol

```diff
@@ -505,22 +498,22 @@ contract Vault is ERC4626, ERC20Permit, ILiquidationSource, Ownable {
    */
   function depositWithPermit(
     uint256 _assets,
+    address _owner,
     address _receiver,
     uint256 _deadline,
     uint8 _v,
     bytes32 _r,
     bytes32 _s
   ) external returns (uint256) {
-    _permit(IERC20Permit(asset()), msg.sender, address(this), _assets, _deadline, _v, _r, _s);
-    return deposit(_assets, _receiver);
+    _permit(IERC20Permit(asset()), _owner, address(this), _assets, _deadline, _v, _r, _s);
+    return _depositAssets(_assets, _owner, _receiver);
```

Comment:
The _receiver isn't checked as part of the signature, which means that anyone can pass an arbitrary `_receiver`.

Reply:
Without a custom signature, I think the only way to do this is have receiver = owner.

Thoughts?

Reply:
> The _receiver isn't checked as part of the signature, which means that anyone can pass an arbitrary `_receiver`.

Ah yes, the signature could be reused to transfer funds to an arbitrary receiver.
Yes, looks like we can't deposit on behalf when using a signature.
Fixed in this commit: https://github.com/GenerationSoftware/pt-v5-vault/pull/20/commits/8cabf072630012a1543a6a575800be12d9910214